feature/profile/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="profile">Profile</string>
```

Comment:
Please check the nomenclature of the strings

Reply:
Hi Meli, thats my fault, there is no nomenclature of the strings yet